webview/src/plots/components/comparisonTable/ComparisonTableRow.tsx

```diff
@@ -39,6 +44,31 @@ export const ComparisonTableRow: React.FC<ComparisonTableRowProps> = ({
     setIsShown(!isShown)
   }
 
+  const updateMultiImgHeight = (img: HTMLImageElement) => {
+    const aspectRatio = img.naturalWidth / img.naturalHeight
+    const width = img.clientWidth
+    const calculatedHeight = Number.parseFloat((width / aspectRatio).toFixed(2))
+    setMultiImgHeight(calculatedHeight)
```

Comment:
Anyone have a better idea on how to accomplish keeping the images the same height?

Reply:
> Are you able to reproduce the broken behaviour from https://github.com/iterative/vscode-dvc/pull/4372 on main with height: 380px; removed from .multiImageWrapper?

Yes, I'm still able to reproduce. Here's a demo:

https://github.com/iterative/vscode-dvc/assets/43496356/3c7ad2d5-bd55-4aea-91f6-c2875421ecb5

Images need to be same height to slider doesn't "jump". 

>  I wanted to see if simply setting img.styles.height here and not using useState worked

Oh, that should work! I'll try it out.

Reply:
Tested setting img.styles.height on the images directly instead of using `useState`! Realized that we'd have to set img state directly on to all images in the row and we'd have to add an detection to see when a new image is added so we could update the height for that image which makes things more complicated.

But we could apply `--img-height` directly on to the multi image row wrapper instead of each individual cell. That way we can avoid using `useState` and not need to watch for image addition. I'll update the code :)